CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
 if(CCPP)
 
+  if(DEBUG)
+    set(_ccpp_debug_arg "--debug")
```

Comment:
where does this get passed to `ccpp_prebuild.py`?

Reply:
That's embarrassing. I fixed it, thanks.